src/lib/auth.js

```diff
@@ -38,14 +38,17 @@ export default authRouter;
  *       default:
  *         $ref: "#/components/responses/default"
  */
-authRouter.post('/login', async (req, res) => {
```

Comment:
The endpoint should be /auth/login because we will also have /auth/register and probably other related endpoints